docs/advanced-usage.rst

```diff
@@ -84,9 +84,10 @@ All transactions need to be signed by the user creating the transaction.
 
     In [0]: signed_tx = bdb.transactions.fulfill(tx, private_keys=alice.private_key)
 
-Write the transaction to the bigchain. The transaction will be stored in a
-backlog where it will be validated, included in a block, and written to the
-bigchain.
+    In [0]: signed_tx
+
+Write the transaction to BigchainDB. The transaction will be stored in a
+backlog where it will be validated before being included in a block.
```

Comment:
The original text was making references to the tables, in particular, to the `bigchain` (blocks) table.